hooks/README.md

```diff
@@ -1,6 +1,6 @@
 # Iroha hooks
 
-For easy development you can copy those hooks after you clone repo. This way you won't forget to generate docs if anything is changed:
+For easier development you can link git hooks after you clone the repo. This way you won't forget to generate docs if anything is changed:
 ```sh
-$ cp hooks/pre-commit.sample .git/hooks/pre-commit
+$ ln -s ../../hooks/pre-commit .git/hooks/pre-commit
```

Comment:
where is current directory?

Reply:
it may seem confusing but ln target is relative to the ln source

Reply:
While — yes the source is relative, I do think that copying is better than linking for new devs. We haven’t updated the pre-commit hook, and most people’s setup don’t need them.

Reply:
ok, I'll revert this to `cp`